Editor/Editors/CinemachineStoryboardEditor.cs

```diff
@@ -123,6 +124,28 @@ public override void OnInspectorGUI()
             rect.x += EditorGUIUtility.labelWidth;
             if (GUI.Button(rect, "Open"))
                 WaveformWindow.OpenWindow();
+
+            EditorGUILayout.Space();
+            sAdvancedFoldout = EditorGUILayout.Foldout(sAdvancedFoldout, "Advanced");
+            if (sAdvancedFoldout)
+            {
+                ++EditorGUI.indentLevel;
+                
+                EditorGUI.BeginChangeCheck();
+                var renderModeProperty = FindProperty(x => x.m_RenderMode);
+                EditorGUILayout.PropertyField(renderModeProperty);
+                EditorGUILayout.PropertyField(FindProperty(x => x.m_SortingOrder));
+                if (renderModeProperty.enumValueIndex != 0)
```

Comment:
should compare the value to the appropriate enum value instead of creating this hidden dependency on the enum order